.github/workflows/unit_test.yml

```diff
@@ -42,13 +42,16 @@ jobs:
         JVM_OPTS:  -Xmx4G -Xms4G -Xss2M
       run: sbt -Denv.type=test clean "test:compile" scalafmtCheckAll
 
+    # Run SlickPlainQueryTest separately from the rest of the unit tests because for some reason, when run together, these
+    # tests will always fail
     - name: Run tests
       env:
         JAVA_OPTS: -Xmx4G -Xms4G -Xss2M -Duser.timezone=UTC -Denv.type=test -Dmysql.host=localhost -Dmysql.port=3307
         JVM_OPTS:  -Xmx4G -Xms4G -Xss2M
       run: |
-         sbt "project core" coverage test coverageReport
-         sbt "project http" coverage test coverageReport
+        sbt "project http" coverage "testOnly org.broadinstitute.dsde.workbench.leonardo.http.db.RuntimeServiceDbQueriesSpec org.broadinstitute.dsde.workbench.leonardo.http.service.RuntimeServiceV2InterpSpec org.broadinstitute.dsde.workbench.leonardo.http.db.RuntimeServiceDbQueriesSpec" coverageReport
```

Comment:
this isn't ideal. But I couldn't find a way to have all tests run in CI yet these tests pass.

By doing this, we're running a few unit tests twice, but we're not losing any coverage